formats/role.jsonnet

```
@@ -49,9 +49,26 @@
         supports_historical_accounts: {
           type: "boolean",
         },
-        seniority :{
+        seniority: {
           type: "integer",
         },
+	whip_organisation: {
```

Looks like you've used tabs instead of spaces. Whilst that's an age-old debate we shouldn't have in a PR, what threw me was that the commas on lines 72 and 73 don't line up, so it looked as though there was one `}` too many (which would be invalid JSON). I then noticed the problem starts on this line - we should be at the same indentation level as the closing `}` on line 54.

I suggest you go through this file and fix up the spacing like this:

```suggestion
        whip_organisation: {
```

---

Argh, this happens every time I do a quick edit in vim and use tabs out of habit 😆